src/redux/catagories/catagories.js

```diff
@@ -0,0 +1,16 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
Please ensure that value of each action type follows the [ducks pattern](https://github.com/erikras/ducks-modular-redux), so it will be something like:   `const CHECK_STATUS = bookstore/categories/CHECK_STATUS`

Reply:
Thanks well noted